elements/pfe-accordion/src/pfe-accordion-panel.scss

```diff
@@ -19,10 +19,9 @@
 .container {
   @include pfe-clearfix;
   @include pfe-panel-container;
-  --pfe-accordion__panel-container--Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 3) #{pfe-local(Padding, $region: base)} calc(#{pfe-local(Padding, $region: base)} * 1.5);
-
+  padding: pfe-local(Padding, $region: panel-container, $fallback: 0 pfe-local(Padding, $region: panel, $fallback: calc(#{pfe-local(Padding, $region: base)} * 3)) pfe-local(Padding, $region: base) pfe-local(Padding, $region: panel, $fallback: calc(#{pfe-local(Padding, $region: base)} * 1.5)));
   :host([disclosure="true"]) & {
-    --pfe-accordion__panel-container--Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 1.5);
```

Comment:
Note, this PR is already solving for the padding-bottom issue on the panel:

https://github.com/patternfly/patternfly-elements/pull/1443/files

Reply:
Ah yes except it's doing it by updating the variable and Adam's PR is doing it by keep the variable empty.  I think this is the preferred approach right?

Reply:
+1^